az-report-ready/systemd/az-report-ready.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Report VM as ready
+After=network.target
+DefaultDependencies=no
```

Comment:
Do we need this? Reading the [man page](https://www.freedesktop.org/software/systemd/man/systemd.unit.html#DefaultDependencies=) it doesn't seem like we do.

> Generally, only services involved with early boot or late shutdown should set this option to no. It is highly recommended to leave this option enabled for the majority of common units..

Reply:
Yup it probably makes sense to remove it here, I was looking to reduce dependencies as far as possible, since a create-vm API call will only return after the unit has been executed and that would delay time-to-pod for a CAA PodVM (But maybe we don't want to wait for the API call to return, so it's not that important).